client/src/components/Allocations/AllocationsModals/AllocationsManageTeamModal/AllocationsManageTeamModal.jsx

```diff
@@ -187,8 +187,13 @@ const AllocationsManageTeamModal = ({
           searchDisable={isLoading}
           onChange={onChange}
           searchResults={search.results}
-          placeholder="Search by username, email, or last name"
+          placeholder=""
         />
+        <div className={styles['help-text']}>
+          <span>
```

Comment:
Is the `<span>` performing a role? Seems superfluous, but maybe there's a style attached (I run away soon to meet a schedule, so I don't check).

Reply:
Assuming no nesting is necessary, I have suggestions for markup semantics and simplification:

```
        />
        <small className={styles['help-text']}>Search by ...</small>
        <div...
```

```
        />
        <i className={styles['help-text']}>Search by ...</i>
        <div...
```

_P.S. These might need a `display: block` in the CSS._

Reply:
Great suggestions, I will make this change now.